modules/tinymce/src/core/test/ts/browser/dom/SelectionQuirksTest.ts

```diff
@@ -64,8 +65,21 @@ describe('browser.tinymce.core.dom.SelectionQuirksTest', () => {
     TinySelections.setSelection(editor, [ 0, 0, 0 ], 0, [ 0, 0 ], 0);
     TinyContentActions.keyup(editor, Keys.left(), { shift: true });
     assertNormalizeCounter(0);
-    TinyContentActions.keyup(editor, 17, { }); // single ctrl
-    assertNormalizeCounter(1);
+    const isMac = Env.os.isMacOS() || Env.os.isiOS();
+    TinyContentActions.keyup(editor, isMac ? 224 : 17, { }); // single ctrl
```

Comment:
I guess you could use the VK key constants here as well. If we decide to keep them that is.

Reply:
Done!